docs/getting-started-on-snowplow-open-source/quick-start-gcp/query-your-data/index.md

```diff
@@ -0,0 +1,139 @@
+
+Some enrichments are _legacy_ and therefore populate your events table. From the above list, these are the _campaign attribution, referer parser_ and _event fingerprint_ enrichments. The UA parser and YAUAA enrichment also add a separate _entity_ to each event (these are also referred to as contexts since they add additional context to the events in your events table).
+
+<Tabs groupId="destination">
+  <TabItem value="postgres" label="Postgres" default>
+
+The contexts are loaded into separate tables: 
+
+- `atomic.com_snowplowanalytics_snowplow_ua_parser_context_1`
+- `atomic.nl_basjes_yauaa_context_1`
+
+Note: you can join these contexts back to your `atomic.events` using `root_id = event_id`.
+
+  </TabItem>
+  <TabItem value="bigquery" label="BigQuery">
+
+The contexts are loaded into separate columns of the events table:
+
+- `contexts_com_snowplowanalytics_snowplow_ua_parser_context_1_0_0`
+- `contexts_nl_basjes_yauaa_context_1_0_3`
+
+
+  </TabItem>
+</Tabs>
+
+**Step 2. Querying your bad data**
+
+In the last section, we sent a test event that would fail to be processed by your pipeline (specifically one that fails to validate against a schema). This is a fundamental aspect of Snowplow; ensuring that only good quality data reaches your stream, lake and warehouse and syphoning off poor quality data so that you have the ability to correct and recover it. 
+
+As the custom `product_view` event passed through your pipeline, the Enrich application fetches the schema for the event. It does this so it can validate that the structure of the event conforms to what was defined up front, therefore ensuring it is of the quality expected.  Since the schema for the `product_view` event doesn't yet exist in your Iglu schema registry, the event failed to validate.
+
+<Tabs groupId="destination">
+  <TabItem value="postgres" label="Postgres" default>
+
+Your `atomic_bad` schema holds events that have failed to be processed by your pipeline. These are called [failed events](/docs/managing-data-quality/failed-events/understanding-failed-events/index.md).
+
+You will see in Postgres that you have a table called `atomic_bad.com_snowplowanalytics_snowplow_badrows_schema_violation_1.`
+
+:::info
+
+You might also see _adapter failure_ failed events in Postgres. Many adaptor failures are caused by bot traffic, so do not be surprised to see some of them in your pipeline. Find out more [here](/docs/managing-data-quality/failed-events/understanding-failed-events/index.md#adaptor-failure).
+
+:::
+
+ </TabItem>
+  <TabItem value="bigquery" label="BigQuery">
+
+:::caution
+
+Currently the Quick Start guide pipelines do not load bad data into BigQuery directly.
+
+:::
+
+To query it, you will need to follow a few additional steps manually:
+
+- Configure the [GCS Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md) to load bad data from the `Bad Rows` topic into GCS.
```

Comment:
@jbeemster or @istreeter could you confirm these steps?

Reply:
So not just bad_rows but also the bad stream.

You have one bad stream coming from enrich and another coming from the bigquery loader.  So you would need a GCS Loader for both.  Generally when we talk about bad data we are talking about enrich.